src/renderer/components/deposit/add/SymDeposit.tsx

```diff
@@ -676,6 +678,25 @@ export const SymDeposit: React.FC<Props> = (props) => {
     )
   }, [asset, isApprovedERC20Token$, needApprovement, oPoolRouter, subscribeIsApprovedState])
 
+  const reloadFeesCheck = useCallback(() => {
```

Comment:
Why another callback, which is just used once in another `useEffect`. BTW: I still get the point about all this logic in `reloadFeesCheck` ...